ibis/expr/types.py

```diff
@@ -48,6 +48,11 @@ def __bool__(self):
 
     __nonzero__ = __bool__
 
+    def _text_repr(self, memo=None):
+        from ibis.expr.format import ExprFormatter
+
+        return ExprFormatter(self, memo=memo).get_result(get_text_repr=True)
```

Comment:
Having a new method seems unnecessary to me, why not add the `get_text_repr` argument to `_repr()` instead?

Reply:
@datapythonista Yeah I agree, not quite sure why I did that